src/argilla/utils/dependency.py

```diff
@@ -130,7 +137,7 @@ def require_version(requirement: str, func_name: Optional[str] = None) -> None:
 _R = TypeVar("_R")
 
 
-def requires_version(requirement: str) -> Callable[[Callable[_P, _R]], Callable[_P, _R]]:
+def requires_version(requirement: Union[str, List[str]]) -> Callable[[Callable[_P, _R]], Callable[_P, _R]]:
```

Comment:
I think I prefer `requires_depedencies` over `requires_version`

```suggestion
def requires_dependencies(requirement: Union[str, List[str]]) -> Callable[[Callable[_P, _R]], Callable[_P, _R]]:
```

Reply:
makes sense the new name :+1: